src/main/java/com/direwolf20/buildinggadgets/common/tools/GadgetUtils.java

```diff
@@ -327,9 +324,9 @@ public static boolean anchorBlocks(EntityPlayer player, ItemStack stack) {
             }
             List<BlockPos> coords = new ArrayList<BlockPos>();
             if (stack.getItem() instanceof GadgetBuilding) {
-                coords = BuildingModes.getBuildOrders(world, player, startBlock, sideHit, stack); //Build the positions list based on tool mode and range
+                coords = BuildingModes.collectPlacementPos(world, player, startBlock, sideHit, stack, startBlock); //Build the positions list based on tool mode and range
```

Comment:
Requires space on the comment

Reply:
Technically I did not wrote this comment, but I will change this one only since I changed the line is was in.